test/EventStore.Client.PersistentSubscriptions.Tests/SubscriptionToAll/replay_parked.cs

```diff
@@ -0,0 +1,37 @@
+using System.Threading.Tasks;
+using Xunit;
+
+namespace EventStore.Client.SubscriptionToAll {
+	public class replay_parked : IClassFixture<replay_parked.Fixture> {
+		private readonly Fixture _fixture;
+		private const string GroupName = nameof(replay_parked);
+
+		public replay_parked(Fixture fixture) {
+			_fixture = fixture;
+		}
+
+		[Fact]
+		public async Task does_not_throw() {
+			await _fixture.Client.ReplayParkedToAllAsync(GroupName, userCredentials: TestCredentials.Root);
+			
+			await _fixture.Client.ReplayParkedToAllAsync(GroupName, numberOfEvents: 100, userCredentials: TestCredentials.Root);
+		}
+
+		[Fact]
+		public async Task throws_when_given_non_existing() {
+			await Assert.ThrowsAsync<PersistentSubscriptionNotFoundException>(() => 
+				_fixture.Client.ReplayParkedToAllAsync("NonExisting", userCredentials: TestCredentials.Root));
+		}
+		
+		[Fact]
+		public async Task throws_when_given_no_credentials() {
+			await Assert.ThrowsAsync<AccessDeniedException>(() => _fixture.Client.ReplayParkedToAllAsync(GroupName));
```

Comment:
**SUGGESTION:** We might also check for the non-root, non-operations credentials.